CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# How to contribute to JuMP
+
+Welcome! This document explains some of the ways you can contribute to JuMP.
+
+## Join the Discourse forum
+
+First up, join the [Discourse forum](https://discourse.julialang.org/c/domain/opt).
+
+The forum is a good place to ask questions about how to use JuMP. You can also
+use the forum to discuss possible feature requests and bugs before raising a
+Github issue (more on this below).
+
+Aside from asking questions, the easiest way you can contribute to JuMP is to
+help answer questions on Discourse!
+
+## File a bug report
+
+Another way to contribute to JuMP is to file [bug reports](https://github.com/JuliaOpt/JuMP.jl/issues/new?template=bug_report.md).
+
+Before you do however, make sure you read the following before posting a new bug
```

Comment:
This repeats the same points from the bug report template. We can refer to that instead.